test/unit/specs/components/governance/__snapshots__/LiProposal.spec.js.snap

```diff
@@ -8,12 +8,23 @@ exports[`LiProposal has the expected html structure 1`] = `
     class="li-proposal__value"
   >
     <h2>
-      Proposal Title
+      <a
+        class=""
+        href="#/governance/1"
+      >
+        Proposal Title
+      </a>
     </h2>
+    <span
+      class="validator-profile__status green"
+    />
     <p>
-      Status: Passed
+      Proposal description…
```

Comment:
Add the `...` only if the length is > 100

Reply:
smart! thanks @fedekunze